ground/gcs/src/plugins/uavsettingsimportexport/importsummary.cpp

```diff
@@ -54,6 +91,15 @@ ImportSummaryDialog::ImportSummaryDialog( QWidget *parent) :
 ImportSummaryDialog::~ImportSummaryDialog()
 {
     delete ui;
+    delete importedObjects;
```

Comment:
Okay, tracked it down a bit further and it looks brittle. `importedObjects` is assigned locally, but it's pointer is `importedObjectManager`, which is created in `uavsettingsimportexportfactory.cpp`. So the `delete` is happening outside the originating class, and the originating class has lost track . This doesn't seem right. Gonna investigate some more.